src/main/java/org/javademos/java21/jep456/UnnamedPatternsAndVariablesDemo.java

```diff
@@ -0,0 +1,65 @@
+package org.javademos.java21.jep456;
+
+import org.javademos.commons.IDemo;
+import java.util.List;
+import java.util.Queue;
+import java.util.LinkedList;
+
+/**
+ * This class demonstrates new features called 'Unnamed Patterns and Variables'.
+ *
+ * REFERENCES:
+ * Java 21 - https://openjdk.org/jeps/456 - Unnamed Patterns and Variables (Preview)
```

Comment:
JEP 456 belongs to Java 22 - https://openjdk.org/jeps/456
JEP 443 is the one included as a preview in Java 21